docs/vms/vms.md

```diff
@@ -68,6 +68,64 @@ vncviewer localhost:<LOCAL_PORT>
 Certain applications, such as Oracle ASM, require a unique identifier for disk drives known as a WWID (World Wide Identifier). In a Linux environment, this can be achieved by utilizing the `ID_PART_ENTRY_UUID` or `ID_PART_ENTRY_NAME` variables. These identifiers can be set in the udev rules file located at `/etc/udev/rules.d/99-asm.rules`. For detailed instructions on configuring disk devices manually for Oracle ASM using WWID, refer to [this guide](https://alexzy.blogspot.com/2018/02/configuring-disk-devices-manually-for.html).
 
 
+### Encrypting VMs
+
+While XCP-ng does not offer native encryption for VMs or storage repositories, there are effective ways to secure your data. 
+
+This guide outlines two common approaches: encrypting data within the VM and using encrypted shared storage repositories (SRs).
+
+:::warning
+
+- The methods described below are suggestions. They are not officially supported by Xen Orchestra.
+- These approaches apply only to VMs and do not cover encryption for [dom0](../guides/dom0-memory.md).
```

Comment:
It's good to provide a link to explain what dom0 is, but the guide about dom0 memory doesn't seem appropriate to me. 

You should improve the glossary, instead: https://docs.xcp-ng.org/appendix/glossary/